src/aura/contactList/contactList.css

```diff
@@ -0,0 +1,6 @@
+.THIS.custom-card{
```

Comment:
same here, you wanna target the custom-card class that is inside the Comp. so you're missing a space there. Also be aware that the margin you are adding those 10px will "break" your layout, first child and last child  of every "row" must loose the corresponding left/right margin